woocommerce/woocommerce
# Add Issue Transfer to Monorepo Merge Tool

### All Submissions:






### Changes proposed in this Pull Request:



This pull request adds a new command to `tools/monorepo-merge`. The command `transfer-issues` allows us to transfer issues from another repository into the monorepo. It features support for adding labels to transferred issues with the `--labels` flag, as well as granular control over the issues using a `--searchFilter` flag.

### How to test the changes in this Pull Request:

1. [Create a GitHub token](https://github.com/settings/tokens).
2. [Create an issue in the smooth generator repository](http://github.com/woocommerce/wc-smooth-generator/issues/new). Any repository would work for this, but to prevent accidents, let's use something Atlas owns. I plan to test the full merge on this repository anyway!
3. Run `tools/monorepo-merge/bin/dev transfer-issues "woocommerce/wc-smooth-generator" --labels "plugin: woocommerce" --searchFilter "is:open author:{YOURGITHUBUSERNAMEHERE}"` after replacing the GitHub username with your own. Confirm that it's only going to transfer a single issue, and if it won't, replace this with a more specific filtering option until you are transferring only a single issue.
4. Verify that the issue has been transferred to the monorepo and that the `plugin: woocommerce` label has been added.

Delete the issue after you're done!

### Other information:




### Changelog entry

N/A

### FOR PR REVIEWER ONLY:
